### PR TITLE
Open sidebar on click on Google Chrome. Issue #10

### PR DESCRIPTION
**Pull request by A13xs**
Now listen touch and click events.

### CODE REVIEW

File: js/leaflet-sidebar.js [hunk 1]
```diff
@@ -122,6 +124,7 @@ L.Control.Sidebar = L.Control.extend({
     },
 
     _onClick: function(e) {
+		e.preventDefault();
```
Comment (Turbo87) [inline js/leaflet-sidebar.js]: could you please fix the indentation here?

**Comment by Turbo87** [comment]
Sorry for the long delay... did you test it on mobile devices too?

**Comment by A13xs** [comment]
Yes, test it on mobile devices.

**Comment by Turbo87** [comment]
I think this was solved by #35 too. Sorry for the long delay!

**Pull request closed without merging**